doc/sphinxdoc/models.rst

```diff
@@ -137,6 +137,102 @@ Models:
 *Note: We provide models operating with a fixed batch size of 64 samples since it was not possible to port the version with dynamic batch size from ONNX to TensorFlow. Additionally, an ONNX version of the model with* `dynamic batch <https://essentia.upf.edu/models/feature-extractors/discogs-effnet/discogs-effnet-bsdynamic-1.onnx>`_ *size is provided.*
 
 
+Discogs-MAEST
+^^^^^^^^^^^^^
+
+Music Audio Efficient Spectrogram Transformer (`MAEST <https://github.com/palonso/MAEST/>`_) trained to predict music style labels using an in-house dataset annotated with Discogs metadata.
+
+Models:
+
+    .. collapse:: ⬇️ <a class="reference external">discogs-maest-30s-pw</a>
+
+        |
+
+            [`weights <https://essentia.upf.edu/models/feature-extractors/maest/discogs-maest-30s-pw-1.pb>`_, `metadata <https://essentia.upf.edu/models/feature-extractors/maest/discogs-maest-30s-pw-1.json>`_]
+
+            Model trained with a multi-label classification objective targeting 400 Discogs styles.
```

Comment:
It is unclear what is the difference between different versions of  `30s` and `10s` models (`pw`, `pw-ts`, `fs`, `dw`). I wonder if this should be explained. For someone who needs to select, which model should be used by default?

Reply:
Addressed in [53148b2](https://github.com/MTG/essentia/pull/1381/commits/53148b2ec27da92a20c53ce4286e274e1ef192ea)